### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by mh292929**
changed broken gem groups streams, add sum channels

### CODE REVIEW

**Comment by MH140000** [comment]
I received a check error. I do not know where the problem comes from. please check it
All streams are safe and without problems.

File: channels/ir.m3u [hunk 1]
```diff
@@ -91,51 +91,38 @@ https://d2e40kvaojifd6.cloudfront.net/stream/24b/playlist.m3u8
...
-https://d2e40kvaojifd6.cloudfront.net/stream/gem_bollywood/playlist.m3u8
-#EXTINF:-1 tvg-id="GEMClassic.ir" tvg-name="GEM Classic" tvg-country="IR" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_az.png?14" group-title="Classic",GEM Classic (540p) [Geo-blocked]
-#EXTVLCOPT:http-referrer=https://www.gemonline.tv/en-US/Live/Index?channelname=gemclassic
-https://d2e40kvaojifd6.cloudfront.net/stream/gem_classic/playlist.m3u8
+#EXTINF:-1 tvg-id="GEMArabia.ir" tvg-name="GEM Arabia" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_arabia.png" group-title="Music",GEM Arabia
+http://216.66.42.47:7777/GemArabia_HD.m3u8
+#EXTINF:-1 tvg-id="GEMAZ.ir" tvg-name="GEM AZ" tvg-country="IR" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_az.png" group-title="Music",GEM AZ
+http://216.66.42.47:7777/GemAZ_HD.m3u8
+#EXTINF:-1 tvg-id="GEMBollywood.ir" tvg-name="GEM Bollywood" tvg-country="IR" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gembollywood.png" group-title="Movies",GEM Bollywood
+https://stream-cdn.gemonline.tv/live/gembollywood/index.m3u8
```
Comment (Dum4G) [inline channels/ir.m3u:99]: All gemonline.tv links need http-referrer to work. See GEM Comedy

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ir.m3u (hunk 1 above)
Comment (Sphinxroot) [reply channels/ir.m3u:99]: only those have changed  all other channel redirect to  main  GEM TV  , and for me  , they still need referrer to work , all other that hav`nt changed  , they still  work but only a short video will be played


River
https://stream-cdn.gemonline.tv/live/river/index.m3u8
Rubix
https://stream-cdn.gemonline.tv/live/rubix/index.m3u8
MAXX
https://stream-cdn.gemonline.tv/live/gemmaxx/index.m3u8
Gem TV
https://stream-cdn.gemonline.tv/live/gem/index.m3u8
Gem Travel
https://stream-cdn.gemonline.tv/live/gemtravel/index.m3u8
Gem Series
https://stream-cdn.gemonline.tv/live/gemseries/index.m3u8
Gem Life
https://stream-cdn.gemonline.tv/live/gemlife/index.m3u8
Gem Kids
https://stream-cdn.gemonline.tv/live/gemkids/index.m3u8
Gem Junior
https://stream-cdn.gemonline.tv/live/gemjunior/index.m3u8
Gem Food
https://stream-cdn.gemonline.tv/live/gemfood/index.m3u8
Gem Fit
https://stream-cdn.gemonline.tv/live/gemfit/index.m3u8
Gem Film
https://stream-cdn.gemonline.tv/live/gemfilm/index.m3u8
Gem Drama
https://stream-cdn.gemonline.tv/live/gemdrama/index.m3u8
Gem Classic
https://stream-cdn.gemonline.tv/live/gemclassic/index.m3u8
Gem Bollywood
https://stream-cdn.gemonline.tv/live/gembollywood/index.m3u8

**Comment by MH140000** [comment]
please review it! it's work

**Review by Sphinxroot — CHANGES_REQUESTED** [review]
all GEM Channel  Please let referrer there because i tested in vlc and kodi and they still need them to work

**Comment by MH140000** [comment]
hi. i can not add this line because i don't no how can i change this file again. please fix. thanks.

**Review by Sphinxroot — COMMENTED** [review]
i put back referrer  i will check the other channel if they migrate to the new server

**Comment by MH140000** [comment]
Currently only these channels are available. I put 3 channels, namely gem az, gem arabia and mifa from another server.

**Review by Dum4G — APPROVED** [review]

**Review by Sphinxroot — APPROVED** [review]

**Pull request merged**